modules/fedimint-ln-client/src/pay.rs

```diff
@@ -563,8 +564,10 @@ impl LightningPayRefund {
 pub struct PayInvoicePayload {
     pub federation_id: FederationId,
     pub contract_id: ContractId,
+    // FIXME: use pruned, privacy friendly version without description etc.
+    /// Invoice containing metadata on how to obtain the preimage
+    pub invoice: lightning_invoice::Bolt11Invoice,
```

Comment:
This breaks decoding, this means this change either needs to get both on `master` and `0.2` or cannot be merged.